events/wol_intrigue_events.txt

```diff
@@ -520,7 +520,7 @@ character_event = {
 				}
 			}
 			
-			3000 = { # Adultery of spouse or consort uncovered
```

Comment:
This is not province ID

Reply:
Fixed in 6d39aee5f58d4d8fbe7faf298e3a9d6e83b76d60.